client/app/src/main/java/com/example/criengine/Adapters/MyBooksAdapter.java

```diff
@@ -85,7 +85,7 @@ public void onClick(View v) {
                 break;
             case "accepted":
                 statusText.setText("Accepted");
-                if (!book.getPotentialBorrower().getHandOffCompelte()) {
+                if (book.getGeolocation() == null) {
```

Comment:
This should be testing `if (book.isConfirmationNeeded) {}` so then the owner and borrower can meet up and do the exchange. Is geoLocation ever supposed to be null?

Reply:
Confirmation has very little to do with location being used. A book only has a geolocation when someone accepts a request (or return) and then puts one in themselves. After that the scan(s) can take place, and after _those_ confirmation is no longer needed. Location will be null a majority of the time, in fact. It makes a lot more sense to me to only allow the ability to add a location when location is not yet specified.

Reply:
The owner has to set the meet up location after the request has been accepted,
since the location will depend on who you're giving it to.
Which means the geolocation will be null till it is accepted. At this point the owner should set the location.
I think this change is right.
```
US 09.01.01
As an owner, I want to specify a geo location on a map of where to receive a book when
I accept a request on the book.
```